packages/components/src/confirm-dialog/stories/index.story.tsx

```diff
@@ -16,13 +16,14 @@ import { ConfirmDialog } from '../component';
 
 const meta: Meta< typeof ConfirmDialog > = {
 	component: ConfirmDialog,
-	title: 'Components (Experimental)/Overlays/ConfirmDialog',
+	title: 'Components/Overlays/ConfirmDialog',
 	id: 'components-experimental-confirmdialog',
```

Comment:
```suggestion
	id: 'components-confirmdialog',
```

We want to remove the `experimental` part from the permalinks, and then add them to the [redirect list here](https://github.com/WordPress/gutenberg/blob/d2fce222ebbbef8dbc56eee1badcfe4ae0df04b0/storybook/manager-head.html#L3) so the old links don't get broken.